tests/unit/test_openstack.py

```diff
@@ -126,6 +126,30 @@
 </VirtualHost>
 """
 
+APACHE2_SSL_CONF_RGW = """
+Listen 433
+<VirtualHost 10.5.2.135:433>
+    ServerName 10.5.100.2
+    SSLEngine on
+    SSLCertificateFile /etc/apache2/ssl/keystone/cert_10.5.100.2
+    SSLCertificateChainFile /etc/apache2/ssl/keystone/cert_10.5.100.2
+    SSLCertificateKeyFile /etc/apache2/ssl/keystone/key_10.5.100.2
+    AllowEncodedSlashes On
+</VirtualHost>
+"""
+
+APACHE2_SSL_CONF_RGW = """
+Listen 433
+<VirtualHost 10.5.2.135:433>
+    ServerName 10.5.100.2
+    SSLEngine on
+    SSLCertificateFile /etc/apache2/ssl/keystone/cert_10.5.100.2
+    SSLCertificateChainFile /etc/apache2/ssl/keystone/cert_10.5.100.2
+    SSLCertificateKeyFile /etc/apache2/ssl/keystone/key_10.5.100.2
+    AllowEncodedSlashes On
+</VirtualHost>
+"""
```

Comment:
looks like APACHE2_SSL_CONF_RGW is duplicated?

Reply:
Git sometimes makes you wonder :-D

Reply:
Duplicated CONF block has been resolved.